src/dataExplorer/components/DataExplorerPage.tsx

```diff
@@ -80,19 +106,23 @@ const DataExplorerPage: FC = () => {
             </div>
           </FeatureFlag>
         )}
-        <Page.ControlBar fullWidth={true}>
-          <Page.ControlBarLeft>
-            <ViewTypeDropdown />
-            <VisOptionsButton />
-          </Page.ControlBarLeft>
-          <Page.ControlBarRight>
-            <TimeZoneDropdown />
-            <SaveAsButton />
-          </Page.ControlBarRight>
-        </Page.ControlBar>
-        <Page.Contents fullWidth={true} scrollable={false}>
-          <DataExplorer />
-        </Page.Contents>
+        {!newDataExplorer && (
+          <Page.ControlBar fullWidth={true}>
+            <Page.ControlBarLeft>
+              <ViewTypeDropdown />
+              <VisOptionsButton />
+            </Page.ControlBarLeft>
+            <Page.ControlBarRight>
+              <TimeZoneDropdown />
+              <SaveAsButton />
+            </Page.ControlBarRight>
+          </Page.ControlBar>
+        )}
+        {!newDataExplorer && (
+          <Page.Contents fullWidth={true} scrollable={false}>
+            <DataExplorer />
+          </Page.Contents>
+        )}
```

Comment:
Wait what? Is this supposed to be a ternary?

Reply:
nope, those are two places that display nothing when the setting is set to true because we haven't written it yet

Reply:
So wouldn't it make more sense to have these be 1 thing that's wrapped by a fragment? Right now it seems like a mistake. It might make it easier to understand as its own component so we just do `!newDataExplorer && <TheWholeShabang />`